x-pack/plugins/security/server/routes/profile/get.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+import type { RouteDefinitionParams } from '../';
+import { wrapIntoCustomErrorResponse } from '../../errors';
+import { createLicensedRouteHandler } from '../licensed_route_handler';
+
+export function defineGetProfileRoute({ router, getProfileService }: RouteDefinitionParams) {
+  router.get(
```

Comment:
__issue:__ I may be missing the context, but it looks like these two endpoints don't perform any authorization checks yet, that means that any user can _theoretically_ (if they can get the UID) retrieve and update profile for any other user, is that correct?

If so and if we don't plan to implement these checks in this PR, I think it'd probably make sense to limit access to superusers only for now. What do you think?

```ts
 options: {
   tags: ['access:accessUserProfile' and 'access:updateUserProfle'],
 },
```